### PR TITLE
corrected bad migration 0047

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: libs/firebase-utils/src/lib/firestoreMigrations/0046.ts [hunk 1]
```diff
@@ -1,23 +1,23 @@
 import { Firestore } from '@blockframes/firebase-utils';
 
 /**
- * Update the screener with fake ref if missing
+ * Update genres on movie documents
```
Comment (phpgeekfr) [inline libs/firebase-utils/src/lib/firestoreMigrations/0046.ts:4]: only changed comment and indent

File: libs/firebase-utils/src/lib/firestoreMigrations/0047.ts [hunk 2]
```diff
@@ -2,16 +2,17 @@ import { Firestore } from '@blockframes/firebase-utils';
 import { runChunks } from '../firebase-utils';
 
 /**
- * Update the screener with fake ref if missing
+ * Replace data.docId by data.eventId
```
Comment (phpgeekfr) [inline libs/firebase-utils/src/lib/firestoreMigrations/0047.ts:5]: only changed comment and indent

File: libs/firebase-utils/src/lib/firestoreMigrations/0048.ts [hunk 3]
```diff
@@ -9,80 +9,80 @@ const antartica = 'antartica' as Territory;
...
-    const orgs = await db.collection('orgs').get();
-
-    const patchKey = (country: Territory) => {
-        if (country === coteDIvoire) {
-            return 'ivory-coast-cote-d-ivoire'
-        } else if (country === antartica) {
-            return 'antarctica'
-        }
-        return country
+  const movies = await db.collection('movies').get();
```
Comment (phpgeekfr) [inline libs/firebase-utils/src/lib/firestoreMigrations/0048.ts:12]: only changed indent

File: libs/invitation/src/lib/+state/invitation.firestore.ts [hunk 4]
```diff
@@ -54,7 +54,6 @@ export function createInvitation(params: Partial<InvitationBase<Date>> = {}): In
     id: '',
     mode: 'invitation',   // We need a default value for backend-function strict mode
     type: 'attendEvent',  // We need a default value for backend-function strict mode
-    eventId: '',
```
Comment (phpgeekfr) [inline libs/invitation/src/lib/+state/invitation.firestore.ts:57]: not needed in createInvitation() because attrib is optionnal in interface

**Review by fritzschoff — APPROVED** [review]

**Pull request merged**